OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 10

# Description
Lint fix for Reports component, this is phase 10.

## Files changed
All js/jsx file in src/components/Reports/ProjectReport/

eslint test:
![CleanShot 2023-10-04 at 00 46 05@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/3e35e815-73e8-4c07-bcc9-6cb657bfc9fd)



# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint "src/components/Reports/ProjectReport/*.js" "src/components/Reports/ProjectReport/*.jsx"`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!
